### PR TITLE
Removed everything with CUT quality of 2.

### PR DESCRIPTION
**Pull request by Kelenius**
#### Summary

```SUMMARY: Infrastructure "Removed all tools and recipes with CUT 2."```

#### Purpose of change

In the current state, this quality is nonsensical. It only exists on the integrated toolset, which makes no sense and one of the mod objects, probably copied from the toolset. And there is only one recipe that uses it, soap shavings, for which a recipe that uses CUT 1 exists (it's slightly slower).

#### Describe the solution

Tools reduced to CUT 1. Recipe removed (there is a second recipe that already uses CUT 1).

#### Describe alternatives you've considered

Adding more recipes and tools that would have this quality is a possibility, but it would require a proper rework. I am unsure what kind of tools and recipes would have/use it. Probably not integrated toolset.

### CODE REVIEW

File: data/json/recipes/recipe_medsandchemicals.json [hunk 1]
```diff
@@ -744,18 +744,6 @@
...
-    "result": "soap_flakes",
-    "id_suffix": "fast_cut",
-    "category": "CC_CHEM",
-    "subcategory": "CSC_CHEM_CHEMICALS",
-    "skill_used": "cooking",
-    "time": "10 m",
-    "autolearn": true,
-    "qualities": [ { "id": "CUT", "level": 2 } ],
-    "components": [ [ [ "soap", 1 ] ] ]
-  },
```
Comment (LaVeyanFiend) [inline data/json/recipes/recipe_medsandchemicals.json]: You have to obsolete the recipe, straight deletion will cause an error on load.

Comment (LaVeyanFiend) [reply data/json/recipes/recipe_medsandchemicals.json]: Add this to json/recipes/recipes_obsolete.json:

```
{
  "type": "recipe",
  "id_suffix": "fast_cut",
  "result": "soap_flakes",
  "obsolete": true
}
```

**Comment by Shodan14** [comment]
I always thought using a higher quality tool than required made it faster.

**Comment by I-am-Erk** [comment]
I'm not sure there's call to remove it so much as expand the list of things that have it. It's odd integrated toolset is the only thing with Cut2, but all that's really needed is a decision on what Cut2 represents and then spreading it to more things. I'd suggest it's for things expressly designed for comfortable, rapid cutting, like kitchen knives

There's some argument for even further  granulation of different types and roles of cutting tools. I'm not sure there's an argument for reducing it, myself. I also just spent several hours sharpening my kitchen knives so I may be a bit biased here

**Comment by Kelenius** [comment]
I have no idea what kind of tools would have that quality and what kind of recipes would need it. It can't be for metal or wood, because they already exist. Butchering ability is also already covered. It would have to be a tool that cuts the same things normal knives do, but "better", and a recipe that needs it.

Besides, removing these qualities here doesn't stop someone that wants to make a cutting tool expansion from adding them where appropriate. There's no reason to keep it as is.

**Comment by mkikt4743** [comment]
could probably use cutting 2 on some higher end knives and make more bone recipes, just removing content from the game seems like a bad way to code

**Comment by Brian-Otten** [comment]
This seems good to me. Between the fine cutting quality already being a way to differentiate cutting recipes and there being some trouble thinking of way to make higher cutting values used this seems like a good idea to merge until the situation changes.

**Comment by sayke** [comment]
What about scissors having cutting 2, and making tailoring recipes require it?

**Comment by I-am-Erk** [comment]
Scissors and tailoring would be another good candidate, but I'd suggest having a quality SHEARS would be a better one there... if just because cut2 implies more an ability to do what knives do but better/faster, while scissors on cloth is really a different application

**Comment by Kelenius** [comment]
This is all marvelous, but outside of the scope of this PR. I'm not removing any content, it's just a small cleanup. There's nothing stopping anyone from adding CUT 2 tools and recipes in a proper way, and they would probably have to make changes from this PR anyway, because it doesn't make sense in the current state.

**Pull request merged**